springql-core/Cargo.toml

```diff
@@ -15,46 +15,35 @@ keywords = ["springql", "stream-processing"] # up to 5 keywords, each keyword sh
 readme = "../README.md"
 repository = "https://github.com/SpringQL/SpringQL"
 
+[features]
+stub_web_console=[]
```

Comment:
Add new feature flag, no default.

This feature enable stub for out going http request to web console.